### PR TITLE
Complete normed vector space

### PR DESCRIPTION
**Pull request by eggrobin**
`PluginAdapter` was renamed since the class name is the `name` of the `SCENARIO` `ConfigNode` in which we write things, it's probably best to have the name `Principia` somewhere there.

### CODE REVIEW

File: ksp_plugin/interface.cpp [hunk 1]
```diff
@@ -360,6 +360,7 @@ double principia__current_time(Plugin const* const plugin) {
 }
 
 char const* principia__SerializePlugin(Plugin const* const plugin) {
+  LOG(INFO) << __FUNCTION__;
```
Comment (pleroy) [inline ksp_plugin/interface.cpp]: VLOG(1), maybe?  (Many places.)

File: ksp_plugin_adapter/ksp_plugin_adapter.cs [hunk 2]
```diff
@@ -6,8 +6,11 @@
 namespace principia {
 namespace ksp_plugin_adapter {
 
-[KSPAddon(startup : KSPAddon.Startup.MainMenu, once : false)]
-public partial class PluginAdapter : UnityEngine.MonoBehaviour {
+[KSPScenario(createOptions: ScenarioCreationOptions.AddToAllGames,
+             tgtScenes: new GameScenes[]{GameScenes.SPACECENTER,
+                                         GameScenes.EDITOR, GameScenes.FLIGHT,
```
Comment (pleroy) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: One parameter per line (unless it's autoformatted by VS).

File: ksp_plugin_adapter/ksp_plugin_adapter.cs [hunk 3]
```diff
@@ -221,29 +228,54 @@ where is_in_physics_bubble(vessel)
...
+  // These functions override virtual ones from |ScenarioModule|, but it seems
+  // that they're actually called by reflection, so that bad things happen
+  // if you don't have, e.g., a function called |OnAwake()| that calls
+  // |base.OnAwake()|.  It doesn't matter whether the functions are public or
+  // private, overriding or hiding though.
+
+  public override void OnAwake() {
+    base.OnAwake();
+    // While we're here, we might as well log.
+    Log.Info("principia.ksp_plugin_adapter.PluginAdapter.OnAwake()");
```
Comment (pleroy) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: PrincipiaPluginAdapter?  (Several places.)

File: ksp_plugin/interface.cpp (hunk 1 above)
Comment (eggrobin) [inline ksp_plugin/interface.cpp]: It's aperiodic (called only on scene load/save) which should be `INFO` per #347.

File: ksp_plugin_adapter/ksp_plugin_adapter.cs (hunk 2 above)
Comment (eggrobin) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Done.

File: ksp_plugin_adapter/ksp_plugin_adapter.cs (hunk 3 above)
Comment (eggrobin) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Done.

**Pull request merged**